<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>land</class>
 <widget class="QDialog" name="land">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="CoinLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Coin2.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="CoinLineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="WorkerLineEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="WorkerLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Worker2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="LandLineEdit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LandLabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/GreenLand2.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GreenLandAddPushButton">
   <property name="geometry">
    <rect>
     <x>664</x>
     <y>80</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WorkerAddPushButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="storePushButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>190</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>store</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="storeWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Wheat</string>
    </attribute>
    <widget class="QPushButton" name="WheatBuy">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Buy</string>
     </property>
    </widget>
    <widget class="QLabel" name="WheatNumberLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
